tools/ProjectTestRunner/NuGet.config

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <clear/>
```

Comment:
is it a relevant change?

Reply:
The pipeline wont run without this due to a security policy check. I'm not sure why it works with CI, but maybe arcade handles that there.